src/common/list/List.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import styels from './list.module.scss';
+
+interface Props {
+  size?: 'small' | 'standard' | 'large';
+  custom?: boolean;
```

Comment:
Could we use `className` prop here instead of `custom` as in `Paragraph.tsx`?

Reply:
Actually, this `custom` prop is not for allowing a custom className; it is a boolean that will strip out the bullet points and all default styles from a list.
Couldn't come up with a better name 😃

Reply:
Alright. Would something like `resetStyle` make sense?